src/xmipp/applications/scripts/tomo_misalignment_resid_statistics/batch_tomo_misalignment_resid_statistics.py

```diff
@@ -0,0 +1,535 @@
+"""
+/***************************************************************************
+ *
+ * Authors:    Federico P. de Isidro Gomez			  fp.deisidro@cnb.csic.es
+ *
+ * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+ * 02111-1307  USA
+ *
+ *  All comments concerning this program package may be sent to the
+ *  e-mail address 'xmipp@cnb.csic.es'
+ ***************************************************************************/
+"""
+
+
+from xml.dom import XML_NAMESPACE
```

Comment:
just checking, do we have this library in the xmipp installer?

Reply:
The script run correctly, honestly I do not remember why it has been added but I am afraid to remove it and break some funcitonality. I can confirm it is defined in the init as:

XML_NAMESPACE: Final = "http://www.w3.org/XML/1998/namespace"